.idea/runConfigurations/All.xml

```diff
@@ -1,21 +1,20 @@
-<component name="ProjectRunConfigurationManager">

```

Comment:
why is this changed? CRLF issues?

Reply:
yes. that and removing the license check. removed that to another commit, but the whitespace fix remains and its tricky to remove so will leave it in.